jellyfin_kodi/helper/utils.py

```diff
@@ -535,3 +536,18 @@ def get_filesystem_encoding():
         enc = 'utf-8'
 
     return enc
+
+
+def find_library(server, item):
+    from database import get_sync
```

Comment:
Need to move import to top of file instead of in function

Reply:
It doesn't work.  No, I don't know why, but it throws import errors in the database init.  Mystery for another day, imo.